pkg/autodiscover/autodiscover_pv.go

```diff
@@ -39,3 +42,17 @@ func getPersistentVolumeClaims(oc corev1client.CoreV1Interface) ([]corev1.Persis
 	}
 	return pvcs.Items, nil
 }
+
+func getAllStorageData() (data []string, err error) {
```

Comment:
"Data" suffix is too generic here. My suggestion: `func getStorageClassProvisioners() ([]string, error)`

Reply:
I think we can leave it as it is.

Reply:
I agree with @greyerof I think this could be more specific to the objects it is retrieving.

Reply:
I changed it. @sebrandon1 @greyerof